### PR TITLE
Update KeyVault documentation

### PR DESCRIPTION
**Pull request by maddieclayton**
## Description
https://github.com/Azure/azure-powershell/issues/6215
https://github.com/Azure/azure-powershell/issues/6034

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/KeyVault/Commands.KeyVault/help/Get-AzureKeyVaultCertificateOperation.md [hunk 1]
```diff
@@ -31,18 +31,28 @@ The **Get-AzureKeyVaultCertificateOperation** cmdlet gets the status of a certif
...
 CancellationRequested     : False
 CertificateSigningRequest : MIICpjCCAY4CAQAwFjEUMBIGA1UEAxMLY29udG9zby5jb20wggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQC73w3VRBOlgJ5Od1PjDh+2ytngNZp+ZP4fkuX8K1Ti5LA6Ih7eWx1fgAN/iTb6l
                             5K6LvAIJvsTNVePMNxfSdaEIJ70Inm45wVU4A/kf+UxQWAYVMsBrLtDFWxnVhzf6n7RGYke6HLBj3j5ASb9g+olSs6eON25ibF0t+u6JC+sIR0LmVGar9Q0eZys1rdfzJBIKq+laOM7z2pJijb5ANqve9
                             i7rH5mnhQk4V8WsRstOhYR9jgLqSSxokDoeaBClIOidSBYqVc1yNv4ASe1UWUCR7ZK6OQXiecNWSWPmgWEyawu6AR9eb1YotCr2ScheMOCxlm3103luitxrd8A7kMjAgMBAAGgSzBJBgkqhkiG9w0BCQ4
                             xPDA6MA4GA1UdDwEB/wQEAwIFoDAdBgNVHSUEFjAUBggrBgEFBQcDAQYIKwYBBQUHAwIwCQYDVR0TBAIwADANBgkqhkiG9w0BAQsFAAOCAQEAIHhsDJV37PKi8hor5eQf7+Tct1preIvSwqV0NF6Uo7O6
                             YnC9Py7Wp7CHfKzuqeptUk2Tsu7B5dHB+o9Ypeeqw8fWhTN0GFGRKO7WjZQlDqL+lRNcjlFSaP022oIP0kmvVhBcmZqRQlALXccAaxEclFA/3y/aNj2gwWeKpH/pwAkZ39zMEzpQCaRfnQk7e3l4MV8cf
                             eC2HPYdRWkXxAeDcNPxBuVmKy49AzYvly+APNVDU3v66gxl3fIKrGRsKi2Cp/nO5rBxG2h8t+0Za4l/HJ7ZWR9wKbd/xg7JhdZZFVBxMHYzw8KQ0ys13x8HY+PXU92Y7yD3uC2Rcj+zbAf+Kg==
-ErrorCode                 : 
+                            ==
```
Comment (cormacpayne) [inline src/ResourceManager/KeyVault/Commands.KeyVault/help/Get-AzureKeyVaultCertificateOperation.md]: @maddieclayton is this line needed?

File: src/ResourceManager/KeyVault/Commands.KeyVault/help/Add-AzureKeyVaultManagedStorageAccount.md [hunk 2]
```diff
@@ -230,11 +261,10 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## INPUTS
 
 ### None
-This cmdlet does not accept any input.
```
[the remark this answers is not in the document]
Comment (maddieclayton) [reply src/ResourceManager/KeyVault/Commands.KeyVault/help/Add-AzureKeyVaultManagedStorageAccount.md]: I didn't think it added more value than "None"

File: src/ResourceManager/KeyVault/Commands.KeyVault/help/Remove-AzureKeyVaultCertificateContact.md [hunk 3]
```diff
@@ -37,11 +37,18 @@ The **Remove-AzureKeyVaultCertificateContact** cmdlet deletes a contact that is
...
+Email               VaultName
+-----               ---------
+user1@microsoft.com mvault2
+user2@microsoft.com mvault2
+user3@microsoft.com mvault2
+user4@microsoft.com mvault2
 ```
 
-This command removes Patti Fuller as a certificate contact for the Contoso01 key vault.
+This command removes Patti Fuller as a certificate contact for the Contoso01 key vault.  If PassThru is specified, the cmdlet returns the list of remaining certificate contacts.
```
[the remark this answers is not in the document]
Comment (maddieclayton) [reply src/ResourceManager/KeyVault/Commands.KeyVault/help/Remove-AzureKeyVaultCertificateContact.md]: Good catch, added -PassThru

File: src/ResourceManager/KeyVault/Commands.KeyVault/help/Backup-AzureKeyVaultKey.md [hunk 4]
```diff
@@ -41,23 +41,29 @@ Use the **Backup-AzureKeyVaultKey** cmdlet to retrieve the key in encrypted form
 ## EXAMPLES
 
 ### Example 1: Back up a key with an automatically generated file name
-```
-PS C:\>Backup-AzureKeyVaultKey -VaultName 'MyKeyVault' -Name 'MyKey'
+```powershell
+PS C:\Users\username\> Backup-AzureKeyVaultKey -VaultName 'MyKeyVault' -Name 'MyKey'
```
[the remark this answers is not in the document]
Comment (RandalliLama) [reply src/ResourceManager/KeyVault/Commands.KeyVault/help/Backup-AzureKeyVaultKey.md]: I was just curious.  I don't have any objections.

File: src/ResourceManager/KeyVault/Commands.KeyVault/help/Backup-AzureKeyVaultCertificate.md [hunk 5]
```diff
@@ -41,23 +41,29 @@ and specify a key vault in the second region.
...
 
 This command retrieves the certificate named MyCert from the key vault named MyKeyVault and saves a backup of that certificate to a file named Backup.blob.
 
 ### Example 3: Back up a previously retrieved certificate to a specified file name, overwriting the destination file without prompting.
-```
-PS C:\>$cert = Get-AzureKeyVaultCertificate -VaultName 'MyKeyVault' -Name 'MyCert'
-PS C:\>Backup-AzureKeyVaultCertificate -Certificate $cert -OutputFile 'C:\Backup.blob' -Force
+```powershell
+PS C:\> $cert = Get-AzureKeyVaultCertificate -VaultName 'MyKeyVault' -Name 'MyCert'
+PS C:\> Backup-AzureKeyVaultCertificate -InputObject $cert -OutputFile 'C:\Backup.blob' -Force
```
Comment (RandalliLama) [inline src/ResourceManager/KeyVault/Commands.KeyVault/help/Backup-AzureKeyVaultCertificate.md]: Should InputObject we changed back to Certificate here?

File: src/ResourceManager/KeyVault/Commands.KeyVault/help/Backup-AzureKeyVaultKey.md [hunk 6]
```diff
@@ -41,23 +41,29 @@ Use the **Backup-AzureKeyVaultKey** cmdlet to retrieve the key in encrypted form
...
 
 This command retrieves the key named MyKey from the key vaultnamed MyKeyVault and saves a backup of that key to a file named Backup.blob.
 
 ### Example 3: Back up a previously retrieved key to a specified file name, overwriting the destination file without prompting.
-```
-PS C:\>$key = Get-AzureKeyVaultKey -VaultName 'MyKeyVault' -Name 'MyKey'
-PS C:\>Backup-AzureKeyVaultKey -Key $key -OutputFile 'C:\Backup.blob' -Force
+```powershell
+PS C:\> $key = Get-AzureKeyVaultKey -VaultName 'MyKeyVault' -Name 'MyKey'
+PS C:\> Backup-AzureKeyVaultKey -InputObject $key -OutputFile 'C:\Backup.blob' -Force
```
Comment (RandalliLama) [inline src/ResourceManager/KeyVault/Commands.KeyVault/help/Backup-AzureKeyVaultKey.md]: Change back to Key?

File: src/ResourceManager/KeyVault/Commands.KeyVault/help/Backup-AzureKeyVaultManagedStorageAccount.md [hunk 7]
```diff
@@ -41,23 +41,29 @@ and specify a key vault in the second region.
...
 
 This command retrieves the managed storage account named MyMSAK from the key vault named MyKeyVault and saves a backup of that managed storage account to a file named Backup.blob.
 
 ### Example 3: Back up a previously retrieved managed storage account to a specified file name, overwriting the destination file without prompting.
-```
-PS C:\>$msak = Get-AzureKeyVaultManagedStorageAccount -VaultName 'MyKeyVault' -Name 'MyMSAK'
-PS C:\>Backup-AzureKeyVaultManagedStorageAccount -StorageAccount $msak -OutputFile 'C:\Backup.blob' -Force
+```powershell
+PS C:\> $msak = Get-AzureKeyVaultManagedStorageAccount -VaultName 'MyKeyVault' -Name 'MyMSAK'
+PS C:\> Backup-AzureKeyVaultManagedStorageAccount -InputObject $msak -OutputFile 'C:\Backup.blob' -Force
```
Comment (RandalliLama) [inline src/ResourceManager/KeyVault/Commands.KeyVault/help/Backup-AzureKeyVaultManagedStorageAccount.md]: Change back to StorageAccount?

**Comment by maddieclayton** [comment]
@RandalliLama Addressed the three comment - if this is looking good to you, I will target it for the release going out on Tuesday morning.

**Comment by RandalliLama** [comment]
:shipit:

**Pull request merged**